geopandas/base.py

```diff
@@ -146,12 +145,12 @@ def crs(self):
         :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`,
         such as an authority string (eg "EPSG:4326") or a WKT string.
         """
-        return self._crs
+        return self.geometry.values.crs
 
     @crs.setter
     def crs(self, value):
         """Sets the value of the crs"""
-        self._crs = None if not value else CRS.from_user_input(value)
+        self.geometry.values.crs = None if not value else value
```

Comment:
```suggestion
        self.geometry.values.crs = value
```

This logic of "None if not value else value" is also done on the array level I suppose?